ipa-core/src/helpers/transport/query/mod.rs

```diff
@@ -226,8 +228,77 @@ impl AsRef<str> for QueryType {
         }
     }
 }
+#[cfg(test)]
+impl PartialEq for IpaQueryConfig {
+    fn eq(&self, other: &Self) -> bool {
+        self.per_user_credit_cap == other.per_user_credit_cap
+            && self.max_breakdown_key == other.max_breakdown_key
+            && self.attribution_window_seconds == other.attribution_window_seconds
+            && self.num_multi_bits == other.num_multi_bits
+            && self.with_dp == other.with_dp
+            && self.epsilon == other.epsilon
+            && self.plaintext_match_keys == other.plaintext_match_keys
+    }
+}
+
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum DpParams {
+    NoDp,
+    WithDp { epsilon: f64 },
+}
+
+impl FromStr for DpParams {
+    type Err = ParseFloatError;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
```

Comment:
if we don't really have tests that cover this, I am a little worried here

Reply:
yeah, since we're not using this code right now, I'll just remove it from this PR.